source/about/membership.md

```diff
@@ -56,6 +73,8 @@ Is your institution a champion of open access and a thought leader in scholarly
 
 - [Newsletter Sign Up](email_sign_up.md)
 - [Blog](https://blog.arxiv.org/)
-- [Twitter](https://twitter.com/arxiv)
+- [Twitter](https://x.com/arxiv)
 - [Status Updates](https://status.arxiv.org/)
-- [Questions? We're happy to help](mailto:membership@arxiv.org)
+- [Questions? We're happy to help](membership@arxiv.org)
```

Comment:
does this no longer need to be an explicit mailto link?

Reply:
This fails in Chrome, despite working in Safari. Suggest replacing the two instances of `(membership@arxiv.org)` to being `(mailto:membership@arxiv.org)`.